recording-oracle/src/modules/campaigns/campaigns.service.ts

```diff
@@ -606,42 +624,37 @@ export class CampaignsService {
         continue;
       }
 
-      /**
-       * !!! NOTE !!!
-       * There can be a situation where two campaign participants
-       * have a trade between each other, so total volume
-       * is not 100% accurate in this case, but probability of it is
-       * negligible so omit it here. Later RepO can verify it if needed.
-       */
-      totalVolume += participantOutcomes.totalVolume;
-
       outcomes.push({
         address: participant.evmAddress,
-        score: participantOutcomes.score,
-        total_volume: participantOutcomes.totalVolume,
+        ...participantOutcomes,
       });
```

Comment:
```suggestion
      } as ParticipantOutcome);
```
Just to silence eslint